node/lib/Logger.ts

```diff
@@ -20,62 +22,89 @@ export class Logger implements ILogger {
             this.globalLogLevel = logLevel;
         }
 
-        let consoleTransport = new winston.transports.Console({
+        this.consoleTransport = new winston.transports.Console({
```

Comment:
We can turn it off using removeTransport(winston.transports.Console);

---
In reply to: [105495262](https://github.com/openT2T/opent2t/pull/38#discussion_r105495262) [](ancestors = 105495262,105340367)